tbot_contrib/gpio/gpio.py

```diff
@@ -0,0 +1,60 @@
+from tbot.machine import linux
+
+
+class gpio:
+    def __init__(self, lnx: linux.LinuxShell, gpio_number: str):
+        self._lnx = lnx
+        self._gpio_number = gpio_number
+        self._gpio_sysclass_path = "/sys/class/gpio/"
+        self._gpio = self._gpio_sysclass_path + "gpio" + self._gpio_number
+        self._export()
+
+    def _export(self) -> None:
+        output = self._lnx.exec0(
+            "find", self._gpio_sysclass_path, "-name", "gpio" + self._gpio_number
+        )
+        if output.strip() == (self._gpio):
+            pass
+        else:
+            self._lnx.exec0(
+                "echo",
+                self._gpio_number,
+                linux.Raw(">"),
+                self._gpio_sysclass_path + "export",
+            )
+
+    def _is_in_direction(self) -> bool:
+        output = self._lnx.exec0("cat", self._gpio + "/direction")
+        if output.strip() == "in":
+            return True
+        else:
+            return False
```

Comment:
There is a [new helper](https://tbot.tools/modules/machine_linux.html#tbot.machine.linux.Path.read_text) which is nicer than a manual `cat`.  Additionally, an `if-else` block which just returns `True` and `False` is superfluous:

```suggestion
        direction = (self._gpio_path / "direction").read_text()
        return direction.strip().lower() == "in"
```